src/sentry/static/sentry/app/components/dropdownMenu.jsx

```diff
@@ -101,7 +101,12 @@ class DropdownMenu extends React.Component {
       onClickOutside(e);
     }
 
-    this.handleClose(e);
+    // Timeout to handle clicks in react portals (e.g. `Hovercard`)
+    // before closing the dropdown. Portals are considered a click "outside"
+    // since they exist outside the DOM hierarchy of `DropdownMenu`
+    window.setTimeout(() => {
+      this.handleClose(e);
+    }, 0);
```

Comment:
I wonder if we turned this function into an async function and used `await Promise.resolve()` if that would look a little cleaner than this /cc @billyvg

Reply:
yeah I could go either way, but I think you're right esp given me previous comment about `timeout`

Reply:
Lets make it into an async function and await a immediately resolved promise, (thus sticking it at the end of the event loop)

Reply:
hm using `await Promise.resolve()` instead doesn't seem to be working on my end